ambry-store/src/main/java/com.github.ambry.store/PersistentIndex.java

```diff
@@ -453,19 +449,19 @@ public IndexValue findKey(StoreKey key, FileSpan fileSpan) throws StoreException
    * @param fileSpan The file span represented by this entry in the log
    * @throws StoreException
    */
-  public void markAsDeleted(StoreKey id, FileSpan fileSpan) throws StoreException {
-    validateFileSpan(fileSpan);
+  void markAsDeleted(StoreKey id, FileSpan fileSpan) throws StoreException {
```

Comment:
can you enlighten me as to how we set the original msg offset for a delete record